### DISCUSSION TITLE
Hyprland didn't start correctly after installation on tumbleweed [Virtual Machine]

### DISCUSSION THREAD

**Discussion by Dolbaeb777K** [end-4/dots-hyprland#541] [Extra Distros]
Hello!
I'm tried to install your **conf** in the _qemu/kvm_ virtual machine on _opensuse_ `tumbleweed`, I used the bash script in instruction for openSUSE. Then I used the manual `installition` in your [wiki](https://end-4.github.io/dots-hyprland-wiki/en/i-i/01setup/) but skiped the **scriptdata/dependencies.conf** point, becuase I thought It installed already.

So, When I typed `Hyprland`, I got the "command no found". I install this by `zypper in hyprland`, then I typed `Hyprland` again
I got the error
I switched to the `tty3` and typed again,
now
I have this:
![изображение](https://github.com/end-4/dots-hyprland/assets/126266582/b71d3a51-7c97-4502-a520-67c43c981376)

What can I to do ?

**Comment by end-4** [comment]
I don't know about opensuse, but here's a note for VM users: https://wiki.hyprland.org/Getting-Started/Master-Tutorial/#vm

**Reply by Dolbaeb777K** [reply]
so, I turned on 3D accel but problem didn't disappear

I have recorded the process:

https://github.com/end-4/dots-hyprland/assets/126266582/5a4af929-37ac-4ffe-a456-0eb1e568069f

What did I do wrong?

**Reply by gitmanbloggus** [reply]
> so, I turned on 3D accel but problem didn't disappear
> 
> I have recorded the process:
> 
> https://github.com/end-4/dots-hyprland/assets/126266582/5a4af929-37ac-4ffe-a456-0eb1e568069f
> 
> What did I do wrong?

start ags manually, if it gives you error 71, we're are both stuck in the same spot
